### PR TITLE
Replaced calls to `random` by `rand`

### PR DESCRIPTION
**Pull request by ariel-anieli**
Hello committers,

Thanks for maintaining this project. I would like to bring back to the community, and also to improve my Erlang skills.

Here is the first of few PRs on clearing out compilation warnings. For this one, [`random` is deprecated since Erlang/OTP 19](https://www.erlang.org/doc/deprecations.html#functions-deprecated-in-otp-19).

Thank you,

### CODE REVIEW

**Comment by prefiks** [comment]
Look good.

**Comment by ariel-anieli** [comment]
> Look good.

Thanks, @prefiks; I'll push other PRs in coming days; [for clearing out warnings in OTP 23](https://github.com/processone/tsung/actions/runs/11911523447/job/33193192433?pr=414).

**Comment by ariel-anieli** [comment]
@prefiks, #416 clears all the warnings in OTP 23; which is the minimal version of the GitHub actions.

**Comment by VlkrS** [comment]
The calls to rand:seed seem wrong, see https://www.erlang.org/doc/apps/stdlib/rand.html#seed/2

There's no seed/3 accepting a state without an algorithm, as implied by the merged code, which results in runtime failures.

At a first glance, when using the rand module instead of random, some seed calls appear gratuitous.

File: src/test/ts_test_config.erl [hunk 1]
```diff
@@ -122,7 +122,7 @@ config_thinktime2_test() ->
     receive
         {timeout,Ref2,end_thinktime} -> ok
     end,
-    random:seed(), % reinit seed for others tests
+    rand:seed(), % reinit seed for others tests
```
Comment (VlkrS) [inline src/test/ts_test_config.erl:125]: Is that even needed? The rand module is autoseeded...

File: src/lib/websocket.erl [hunk 2]
```diff
@@ -131,7 +131,7 @@ decode(Data) ->
 %%% Internal functions
 %%%===================================================================
 gen_accept_key() ->
-    random:seed(erlang:now()),
+    rand:seed(erlang:now()),
```
Comment (VlkrS) [inline src/lib/websocket.erl:134]: What's the point of trying to seed here? crypto:strong_rand_bytes/1 takes care of that, no?

**Comment by ariel-anieli** [comment]
I am just reading your feedback, @VlkrS; it is much appreciated.
Could we sync up? I see you already opened some PRs. I would like to know:

1. Which issues have you already addressed?
2. Which ones would like me to look into?

I will then push the fixes.

**Pull request merged**